backend/src/services/episode/episode.service.ts

```diff
@@ -22,13 +30,33 @@ class Episode {
   public async getById(id: string): Promise<TEpisode> {
     const episode = await this.#episodeRepository.getById(id);
     if (!episode) {
-      throw new HttpError({ status: HttpCode.NOT_FOUND, message: ErrorMessage.NOT_FOUND });
+      throw new HttpError({
+        status: HttpCode.NOT_FOUND,
+        message: ErrorMessage.EPISODE_NOT_FOUND,
+      });
     }
     return episode;
   }
 
-  public create(payload: EpisodeCreatePayload): Promise<TEpisode> {
-    return this.#episodeRepository.create(payload);
+  public async create(payload: EpisodeCreatePayload): Promise<TEpisode> {
+    const episode = await this.#episodeRepository.create({
+      [EpisodeCreatePayloadKey.NAME]: payload[EpisodeCreatePayloadKey.NAME],
+      [EpisodeCreatePayloadKey.USER_ID]:
+        payload[EpisodeCreatePayloadKey.USER_ID],
+      [EpisodeCreatePayloadKey.PODCAST_ID]:
+        payload[EpisodeCreatePayloadKey.PODCAST_ID],
+      [EpisodeCreatePayloadKey.TYPE]: payload[EpisodeCreatePayloadKey.TYPE],
+      [EpisodeCreatePayloadKey.DESCRIPTION]:
+        payload[EpisodeCreatePayloadKey.DESCRIPTION],
+    } as EpisodeCreatePayload);
```

Comment:
```suggestion
    const episode = await this.#episodeRepository.create({
      name: payload[EpisodeCreatePayloadKey.NAME],
      [EpisodeCreatePayloadKey.USER_ID]:
        payload[EpisodeCreatePayloadKey.USER_ID],
      [EpisodeCreatePayloadKey.PODCAST_ID]:
        payload[EpisodeCreatePayloadKey.PODCAST_ID],
      [EpisodeCreatePayloadKey.TYPE]: payload[EpisodeCreatePayloadKey.TYPE],
      [EpisodeCreatePayloadKey.DESCRIPTION]:
        payload[EpisodeCreatePayloadKey.DESCRIPTION],
    });
```
since `episodeRepository.create` is typed, we can avoid computed properties